src/com/epam/javacourse/homework4/HousePlant.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.javacourse.homework4;
+
+public class HousePlant extends Plant {
+
+    private final boolean needsSunlight;
+
+    public HousePlant(String name, String type, String nativeRegion, boolean needsSunlight) {
+        super(name, type, nativeRegion);
+        this.needsSunlight = needsSunlight;
+    }
+
+    @Override
+    public String getPlantType() {
+        return "House Plant";
+    }
+
+    @Override
+    public String getNativeRegion() {
+        return super.getNativeRegion();
+    }
+
+    @Override
+    public String getPlantName() {
+        return super.getPlantName();
+    }
+
+    @Override
+    public void water() {
+        System.out.println("Watering " + getPlantName() + ", a flowering plant");
+    }
+
+    public boolean isNeedsSunlight() {
```

Comment:
This is a getter method, so, please, start the name with `get`.

Reply:
Information about sunlight is required but never used.
In this case, you can override `toString` method from the `Plant` class and add this specific for the `HousePlant` information in it.

Reply:
Done